tmc-prototype/DishMaster/DishMaster/DishMaster.py

```diff
@@ -85,27 +87,29 @@ def increment_position(self,argin):
             temp1 = abs(self._elevation_difference)
         else:
             temp1 = temp
-            pass
 
         if temp == temp1:
             temp = temp + 1
-        else:
-            pass
 
-        for position in range(0,temp):
-            self.set_status("Dish is pointing towards the desired coordinates.")
-            self.devlogmsg("Dish is pointing towards the desired coordinates.", 4)
+        for position in range(0, temp):
+            self.set_status(CONST.STR_DISH_POINT_INPROG)
+            self.devlogmsg(CONST.STR_DISH_POINT_INPROG, 4)
             self._pointing_state = 1
             time.sleep(2)
-            if (self._achieved_pointing[1] == self._desired_pointing[1]) and (self._achieved_pointing[2] == self._desired_pointing[2]):
+            if (self._achieved_pointing[1] == self._desired_pointing[1])\
```

Comment:
Same as above.

Reply:
Ensure line length of 110 characters and split the line with parenthesis if exceeding. Commit id is 172675f.